blog/content/post/2019-06-03-Hummingboard-Pulse-first-impression.md

```diff
@@ -0,0 +1,216 @@
+
+```shell
+umount /dev/sd[x]
+```
+Then you can easily flash your SD card with u-boot simply typing:
+
+```shell
+sudo dd if=flash.bin of=/dev/sd[x] bs=1024 seek=33
+```
+Where [x] stands for your SD card attached to PC. For me it was sdb.
+
+From now you will have working U-Boot on your SD card. If you want to check how
+it looks on HummingBoard Pulse you need to connect your PC with the board using
+MicroUSB to USB cable and minicom on the terminal. Just type:
+
+```shell
+sudo minicom -b 115200 -D /dev/ttyUSB[x]
+```
+
+This time [x] stands for the USB port you use to communicate. For me, it was 0.
+
+You should receive something like this:
+
+```shell
+-Boot SPL 2018.11-00078-g0dd51748c2a (Dec 16 2018 - 18:35:18 +0100)
+PMIC:  PFUZE100 ID=0x10
+Normal Boot
+Trying to boot from MMC2
+NOTICE:  Configureing TZASC380
+NOTICE:  BL31: v1.6(release):v1.6-110-g0eb2df45
+NOTICE:  BL31: Built : 13:56:07, Nov 29 2018
+NOTICE:  sip svc init
+
+
+U-Boot 2018.11-00078-g0dd51748c2a (Dec 16 2018 - 18:35:18 +0100)
+
+CPU:   Freescale i.MX8MQ rev2.0 at 1000 MHz
+Reset cause: POR
+Model: SolidRun i.MX8MQ HummingBoard Pulse
+DRAM:  3 GiB
+MMC:   FSL_SDHC: 0, FSL_SDHC: 1
+Loading Environment from MMC... *** Warning - bad CRC, using default environment
+
+In:    serial
+Out:   serial
+Err:   serial
+Net:
+Error: ethernet@30be0000 address not set.
+
+Error: ethernet@30be0000 address not set.
+eth-1: ethernet@30be0000
+Hit any key to stop autoboot:  0
+```
+
+OK, so now you need to copy the kernel image and .dtb file. Unfortunately, I was not
+able to compile image following instructions given by SolidRun. Happily, there is
+another way to run Linux on the HummingBoard Pulse.
+
+#### Installing Debian on HummingBoard Pulse
+
+SolidRun gives instructions on how to install Debian on HummingBoard Pulse. Debian
+is well-documented GNU/Linux distribution and images of it are easily available
+at https://images.solid-build.xyz/IMX8/Debian/. When you will be choosing one for
+you please check the log of changes at the bottom. You will read there that only
+versions released after 07.12.2018 supports booting from SD card.
+
+After you download and extract an image of Debian you need to flash it on SD card.
+
+```Shell
+dd bs=4k conv=fsync if=<image name>.img of=/dev/sdb
+```
+
+Below you can watch log from booting Debian on HummingBoard Pulse. I capture it
+using asciinema.
+
+[![asciicast](https://asciinema.org/a/250003.svg)](https://asciinema.org/a/250003?speed=1)
+
+## Summary
+
+This post describes the HummingBoard Pulse and attempts of booting a Linux on it.
+If you are looking for more detailed information about the board you are welcome
+to check [this](https://developer.solid-run.com/products/hummingboard-pulse/) or
+[that](https://www.nxp.com/products/processors-and-microcontrollers/arm-based-processors-and-mcus/i.mx-applications-processors/i.mx-8-processors/i.mx-8m-family-armcortex-a53-cortex-m4-audio-voice-video:i.MX8M).
+
```

Comment:
@TomaszAIR At the end of the post please outline the future plans and encourage the reader to leave feedback. Something like:

This is our first post from the i.MX8 series and we hope to work with this platform much more.
Please let us know which i.MX8 features you like the most and what kind of content would you
expect in the next posts.